Makefile

```diff
@@ -34,7 +34,7 @@ export GOPROXY
 export GO111MODULE=on
 
 # Kubebuilder.
-export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.31.0
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.32.0
```

Comment:
Since this is getting bumped, would we want to bump the k8s version in the go.mod as well?

Reply:
Good point, seems like they should be synced ideally. But CAPI has this in the [release-1.9 branch](https://github.com/kubernetes-sigs/cluster-api/blob/release-1.9/go.mod) and still has v0.31.3 in `go.mod` for Kubernetes imports. :shrug:

Reply:
Hmm, interesting 🤷🏻‍♂️